src/pybind/mgr/ansible/ansible_runner_svc.py

```diff
@@ -13,6 +13,8 @@
 PLAYBOOK_EXEC_URL = "api/v1/playbooks"
 PLAYBOOK_EVENTS = "api/v1/jobs/%s/events"
 EVENT_DATA_URL = "api/v1/jobs/%s/events/%s"
+URL_MANAGE_GROUP = "api/v1/groups/{group_name}"
```

Comment:
Why some of these are here and others in `src/pybind/mgr/ansible/module.py` (like `URL_GET_HOST_GROUPS = "api/v1/hosts/{host_name}"`)?